tests/ui-toml/large_include_file/large_include_file.rs

```diff
@@ -0,0 +1,11 @@
+#![warn(clippy::large_include_file)]
+
+// Good
+const GOOD_INCLUDE_BYTES: &[u8; 354] = include_bytes!("large_include_file.rs");
+const GOOD_INCLUDE_STR: &str = include_str!("large_include_file.rs");
+
+// Bad
+const TOO_BIG_INCLUDE_BYTES: &[u8; 571] = include_bytes!("too_big.txt");
+const TOO_BIG_INCLUDE_STR: &str = include_str!("too_big.txt");
+
```

Comment:
Since this is dealing with macros, could you add an explicit test for the allow attribute like this:

```rs
#[allow(clippy::large_include_file)]
const ALLOWED_TOO_BUG_INCLUDE: &str = include_str!("too_big.rs");
```

I'm 99.9% sure that this works without a problem :upside_down_face: